### ISSUE TITLE
关于cutHiveMeshWithPoses方法mask旋转180度的问题

### ISSUE THREAD

**Issue by JKK-dr** [DRosemei/RoMe#52]
mask = mask[::-1, ::-1]
在看源码时，发现这里对mask进行了180度的选择，请问为什么要这么设计呢？

**Comment by DRosemei** [comment]
@JKK-dr You can uncomment [this](https://github.com/DRosemei/RoMe/blob/main/utils/geometry.py#L116) and compate it with BEV images. You will see its shape is similar to the BEV image.

**Comment by JKK-dr** [comment]
Yes, I have tried it. I need to project the vertex coordinates onto the image here. I used NuscDataset's sample["world2camera"] to transform it into the camera coordinate system, and then projected it onto the image based on the camera intrinsics provided by nuScenes. However, I found that they were flipped by 180 degrees. I directly used points[i, 0] = width - points[i, 0], points[i, 1] = height - points[i, 1] to flip them. But I found there is still some offset. Can you help me solve this problem? Thank you very much for your open-source contribution.
